test/functional/p2p_segwit.py

```diff
@@ -1296,9 +1296,9 @@ def test_tx_relay_after_segwit_activation(self):
 
         # Node will not be blinded to the transaction
         self.std_node.announce_tx_and_wait_for_getdata(tx3)
-        test_transaction_acceptance(self.nodes[1], self.std_node, tx3, True, False, 'tx-size')
+        test_transaction_acceptance(self.nodes[1], self.std_node, tx3, True, False)
```

Comment:
Took me forever to remember the issue here -- the first string check can stay, but the second one has to be removed because now that we cache failure of wtxid's in the recentRejects filter, the reason for failure on the second validation attempt will be different.

Reply:
Reinstating this first string check (and the second line was removed in a later commit anyway).